docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '3'
+
+services:
+  meter:
+    build:
+      context: ./
+      dockerfile: ./dockerfile
+    volumes:
```

Comment:
Shouldn't this be `devices`?

Reply:
Dont know if that is the proper way to mount that, it is essentially a file right? / acts that way